tests/sqlparser_common.rs

```diff
@@ -2202,6 +2202,7 @@ fn parse_create_sources() {
     let sql = "CREATE SOURCES FROM 'kafka://whatever' USING SCHEMA REGISTRY 'http://foo.bar:8081'";
     match verified_stmt(sql) {
         Statement::CreateSources {
+            like,
```

Comment:
I'd expect something to complain that this is unused. I.e., you probably want an `assert!(like.is_none())` below.